src/components/Home.jsx

```diff
@@ -57,6 +75,106 @@ function Home() {
           <p>{homeData?.hours_teaching} + hours of teaching</p>
         </div>
       </div>
+      <div className="card">
+        <HStack spacing="24px" mt={10}>
+          <Card
```

Comment:
The card for 'Mission' has a bit of y-offset. Please fix that. 
Design looks good!